webapp/src/components/modals/channel_settings/edit_channel_settings.tsx

```diff
@@ -449,12 +450,22 @@ export default class EditChannelSettings extends PureComponent<Props, State> {
         const enableSubmitButton = Boolean(this.state.filters.projects[0]);
         const enableDeleteButton = Boolean(this.props.selectedSubscription);
 
+        let saveSubscriptionButtonText = 'Save Subscription';
+        if (this.props.creatingSubscription) {
+            saveSubscriptionButtonText = 'Add Subscription';
+        }
+
+        let headerText = 'Edit Jira Subscription for ';
+        if (this.props.creatingSubscription) {
```

Comment:
Can we move these checks into one if statement? Other than that, LGTM!

Reply:
Sure i will